src/bip39.cpp

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright (c) 2013-2014 Tomas Dzetkulic
+ * Copyright (c) 2013-2014 Pavol Rusnak
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining
+ * a copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+ * OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ * THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES
+ * OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+ * ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ * OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+// Source:
+// https://github.com/trezor/trezor-crypto
```

Comment:
Unless im wrong,i dont see the whole repo being depricated,more so they moved to a new repo.Since i didnt see deprication mentioned anywhere on their repo.

Reply:
Checked their new repo,it uses mostly the same code in bip39 if im not mistaken,but in c.

Reply:
If the repository was moved to other place, not updated anymore, and the code was ported to another language plus some changes were done. Then the code is deprecated. The "mostly the same code" in this key creation code is not enough.

If we are going to back port code from other place, we need to ensure that it's the latest one to not end up having problems that were already solved in other place.

Sorry if I sounds hard but this piece of code is in charge of the master key creation, we must go secure in this area.

Reply:
![image](https://user-images.githubusercontent.com/25401217/73296099-152bdb00-421a-11ea-8923-b9546b0fe656.png)
These were the changes,What they (trezor) have done is move those files to a new repo and refactored bip39 code,other than that i dont see any changes.

Reply:
What is important is what it's inside of those commits and where they moved from cpp to c code (what changes they did there and how that changes this code -- if it changes it or not.)

Reply:
It seems that there is no previous history. Instead of refactor the file name, they created this .c as a new one.